logbook-servlet/src/main/java/org/zalando/logbook/servlet/MimeTypes.java

```diff
@@ -0,0 +1,19 @@
+package org.zalando.logbook.servlet;
+
+import lombok.SneakyThrows;
+
+import javax.activation.MimeType;
+import javax.activation.MimeTypeParseException;
+
+final class MimeTypes {
+
+    MimeTypes() {
+        // package private so we can trick code coverage
+    }
+
+    @SneakyThrows(MimeTypeParseException.class)
```

Comment:
I am not sure it is a good idea to fail in case of invalid content type. Should it return `Optional<MimeType>` instead?

Reply:
Done